### PR TITLE
#13 move tests to postgres

### LINKED ISSUE #13: move tests to postgres usage

### PR DESCRIPTION
**Pull request by marsofandrew**
Close #13

### CODE REVIEW

File: src/test/resources/application.yml [hunk 1]
```diff
@@ -5,20 +5,17 @@ server:
 
 spring:
   datasource:
-    url: jdbc:h2:mem:variant-annotator-test
-    username: sa
-    password: sa
-    driverClassName: org.h2.Driver
-    schema: classpath:init.sql
+    schema: classpath:V1__init.sql
+    url: jdbc:tc:postgresql:9.6.8:///test-db
```
Comment (lapter57) [inline src/test/resources/application.yml]: We use postgres:13.2-alpine. I think we should use it in tests too: `jdbc:tc:postgresql:13.2-alpine:///test-db`

File: src/test/resources/application.yml [hunk 2]
```diff
@@ -5,20 +5,17 @@ server:
 
 spring:
   datasource:
-    url: jdbc:h2:mem:variant-annotator-test
-    username: sa
-    password: sa
-    driverClassName: org.h2.Driver
-    schema: classpath:init.sql
+    schema: classpath:V1__init.sql
```
Comment (lapter57) [inline src/test/resources/application.yml]: I think we can remove this property

**Review by Kirvolque — APPROVED** [review]

File: src/test/resources/application.yml (hunk 1 above)
Comment (marsofandrew) [reply src/test/resources/application.yml]: Agree

File: src/test/resources/application.yml (hunk 2 above)
Comment (marsofandrew) [reply src/test/resources/application.yml]: Agree

Comment (marsofandrew) [reply src/test/resources/application.yml:8]: Agree

**Review by lapter57 — APPROVED** [review]

**Pull request merged**